client/lib/purchases/index.js

```diff
@@ -266,7 +266,7 @@ function isRefundable( purchase ) {
  * @returns {boolean} true if the purchase can be removed, false otherwise
  */
 function isRemovable( purchase ) {
-	if ( isRefundable( purchase ) ) {
+	if ( isRefundable( purchase ) && purchase.refundAmount !== 0 ) {
```

Comment:
Why not `> 0` like the others (that seems like it would be less fragile)?

Related: Since this check is being repeated many places, perhaps it would be nice to add a helper function?  Something like:

```
function hasAmountAvailableToRefund( purchase ) {
	if ( isRefundable( purchase ) && purchase.refundAmount > 0 ) {
		return true;
	}

	return false;
}

```

Reply:
Ended up creating a `hasAmountAvailableToRefund` function to avoid repetition. Good suggestion!